folium/features.py

```diff
@@ -753,6 +753,8 @@ class CircleMarker(Marker):
         use Circle.
     color: str, default 'black'
         The color of the marker's edge in a HTML-compatible format.
+    weight: int, default 2
```

Comment:
`Leaflet` docs are hard to follow sometimes. It says that `CircleMarker` extends `Circle` that extends `Path` which has a `weight` default of `5`. What is the reason that lead you to choose `2` instead? (Just curios, you don't need to change to `5`. Images maybe help us decided here :smile: )